third_party/color_emoji/emoji_builder.py

```diff
@@ -353,6 +353,21 @@ def write_bitmapSizeTable (self, glyph_maps):
 		self.pop_stream ()
 
 
+# copied from nototools/font_data
+_UNICODE_CMAPS = {(4, 0, 3), (4, 3, 1), (12, 3, 10)}
+
+def delete_from_cmap(font, chars):
+    """Delete all characters in a list from the cmap tables of a font."""
+    cmap_table = font['cmap']
+    for table in cmap_table.tables:
+        table_tup = (table.format, table.platformID, table.platEncID)
+        if table_tup in _UNICODE_CMAPS:
```

Comment:
If char is a unicode value, and table.cmap is not a unicode cmap, what would happen?  It might be possible that char would accidentally map some unrelated value in the other cmap and remove it.  I think we need to restrict ourselves to unicode cmaps.